### PR TITLE
Changed some styling, aligned the icon and text

### PR DESCRIPTION
**Pull request by Dhruv-Rathi**
@rjvim please review.

### CODE REVIEW

File: src/Base/index.js [hunk 1]
```diff
@@ -36,42 +37,58 @@ const Input = ({
     }
 
     if (focused) {
-      return props.focusColor || '#52525B';
+      return props.focusColor || props.borderColor || '#52525B';
```
Comment (rjvim) [inline src/Base/index.js]: @Dhruv-Rathi I am not understand why we need two props and how do write documentation for this if another developer has to use. Why borderColor is needed, focusColor is enough right?

File: src/Base/index.js [hunk 2]
```diff
@@ -36,42 +37,58 @@ const Input = ({
...
-            borderRadius: 4,
-            paddingHorizontal: 5,
-            marginTop: 5,
-            alignItems: icon ? 'center' : 'baseline',
+            height: props.height || 42,
+            borderWidth: props.borderWidth || 1,
+            borderRadius: props.borderRadius || 4,
+            paddingHorizontal: props.paddingHorizontal || 5,
+            marginTop: props.marginTop || 5,
+            alignItems: 'icon' ? 'center' : 'baseline',
```
Comment (rjvim) [inline src/Base/index.js]: 'icon' in quotes is not clear, what does it do here?

**Review by rjvim — CHANGES_REQUESTED** [review]

File: src/Base/index.js (hunk 1 above)
Comment (Dhruv-Rathi) [reply src/Base/index.js]: @rjvim if there is no focus colour but it is focused then the borderColor should be props.borderColor.

Comment (rjvim) [reply src/Base/index.js]: @Dhruv-Rathi Let's remove on of them. But i guess the terminology "focus" tells that that color comes when user focuses. I think we should call it - `focusBorderColor` -> merge them both plz

File: src/Base/index.js (hunk 2 above)
Comment (rjvim) [reply src/Base/index.js]: @Dhruv-Rathi Plz check above comment too

**Comment by Dhruv-Rathi** [comment]
@rjvim Please check now.

**Pull request merged**